lib/Test/Nginx/Socket.pm

```diff
@@ -1226,6 +1228,25 @@ sub fmt_str ($) {
     $str;
 }
 
+sub check_response_filters ($$$$$) {
+    my ($block, $res, $dry_run, $req_idx, $need_array) = @_;
+    my $name = $block->name;
+
+    if ( defined $block->response_body_filters )
+    {
+        my $content = $res ? $res->content : undef;
+        my $expected = get_indexed_value($name, $block->response_body_filters,
```

Comment:
I think instead of allowing each different request to have a different filter, we can just support a list of filters applying to *all* the requests.